packages/components/src/components/icon/common/ic-filter-dark.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
can we optimize this with svgo?

Reply:
after using the theme function, we don't need this svg and will proceed to remove it